### PR TITLE
add get news

### PR DESCRIPTION
**Pull request by actimellka**

### CODE REVIEW

File: src/DepartmentService.Mappers/Models/DepartmentUserInfoMapper.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    {
+      if (userInfo == null)
+      {
+        return null;
+      }
+
+      return new DepartmentUserInfo
+      {
+        User = userInfo,
+        IsActive = dbDepartmentUser.IsActive,
```
Comment (grinakate) [inline src/DepartmentService.Mappers/Models/DepartmentUserInfoMapper.cs:32]: maybe dbDepartmentUser?.IsActive ?

File: Common.Configurations.targets [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
+    <DebugType>none</DebugType>
+    <Optimize>true</Optimize>
+    <DefineConstants>RELEASE;TRACE</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.53" />
+    <PackageReference Include="LT.DigitalOffice.Kernel.RedisSupport" Version="1.0.2" />
+    <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.1" />
+    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.54" />
```
Comment (MariyaSin) [inline Common.Configurations.targets:29]: please, update models to 57version. In 57 version updated project info model

File: src/DepartmentService.Business/Department/GetDepartmentCommand.cs [hunk 3]
```diff
@@ -1,304 +1,382 @@
...
+    private readonly IRequestClient<IGetImagesRequest> _rcImages;
+    private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsersData;
+    private readonly IRequestClient<IGetNewsRequest> _rcGetNews;
+    private readonly IRequestClient<IGetProjectsRequest> _rcGetProjects;
+    private readonly IRequestClient<IGetPositionsRequest> _rcGetPositions;
+    private readonly IConnectionMultiplexer _cache;
+    private readonly IResponseCreator _responseCreator;
+    
+    
+
```
Comment (MariyaSin) [inline src/DepartmentService.Business/Department/GetDepartmentCommand.cs:59]: remove empty lines

File: src/DepartmentService.Data/DepartmentRepository.cs [hunk 4]
```diff
@@ -1,171 +1,176 @@
...
+      }
+
+      if (filter.IncludeProjects)
+      {
+        dbDepartments = dbDepartments.Include(d => d.Projects.Where(p => p.IsActive));
+      }
+
+      if (filter.IncludeNews)
+      {
+        dbDepartments = dbDepartments.Include(d => d.News.Where(p => p.IsActive));
```
Comment (MariyaSin) [inline src/DepartmentService.Data/DepartmentRepository.cs:62]: p means Project

File: src/DepartmentService.Mappers/Models/DepartmentUserInfoMapper.cs [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    {
+      _positionInfoMapper = positionInfoMapper;
+    }
+
+    public DepartmentUserInfo Map(
+      UserInfo userInfo,
+      DbDepartmentUser dbDepartmentUser,
+      PositionData positionData)
+    {
+      if (userInfo == null)
```
Comment (MariyaSin) [inline src/DepartmentService.Mappers/Models/DepartmentUserInfoMapper.cs:24]: split with next condition

File: src/DepartmentService.Models.Dto/Models/ProjectInfo.cs [hunk 6]
```diff
@@ -1,13 +1,14 @@
...
+namespace LT.DigitalOffice.DepartmentService.Models.Dto.Models
+{
+  public record ProjectInfo
+  {
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public string Status { get; set; }
+    public string ShortName { get; set; }
+    public string ShortDescription { get; set; }
+    public ImageInfo AvatarImage { get; set; }
```
Comment (MariyaSin) [inline src/DepartmentService.Models.Dto/Models/ProjectInfo.cs:12]: Project info contains avatar?

File: src/DepartmentService.Models.Dto/Models/UserInfo.cs [hunk 7]
```diff
@@ -1,19 +1,13 @@
...
-    public DepartmentUserRole Role { get; set; }
-    public ImageInfo AvatarImage { get; set; }
-    public PositionInfo Position { get; set; }
-  }
-}
+﻿using System;
+
+namespace LT.DigitalOffice.DepartmentService.Models.Dto.Models
+{
+  public class UserInfo
```
Comment (MariyaSin) [inline src/DepartmentService.Models.Dto/Models/UserInfo.cs:5]: record

File: src/DepartmentService.Business/Department/FindDepartmentCommand.cs [hunk 8]
```diff
@@ -1,241 +1,243 @@
...
+          : null;
+
+        response.Body.Add(
+          _departmentMapper.Map(
+            dbDepartment,
+            userData == null ?
+              null :
+              _departmentUserMapper.Map(
+                _userMapper.Map(userData, imagesData.FirstOrDefault(i => i.ImageId == userData.ImageId)),
+                dbDepartment?.Users.FirstOrDefault(du => du.UserId == userData.Id),
```
Comment (MariyaSin) [inline src/DepartmentService.Business/Department/FindDepartmentCommand.cs:231]: dbDepartment can not be null, please remove '?' dbDepartment.Users.FirstOrDefault(du => du.UserId == userData.Id)

File: src/DepartmentService.Business/Department/GetDepartmentCommand.cs [hunk 9]
```diff
@@ -1,304 +1,380 @@
...
+      List<DepartmentUserInfo> departmentUsersInfo = null;
+
+      if (usersData != null && usersData.Any())
+      {
+        List<PositionData> positionsData = await GetPositionsAsync(
+          usersData.Select(u => u.Id).Distinct().ToList(),
+          response.Errors);
+
+        List<ImageData> imagesData = await GetUserImagesAsync(
+          usersData.Where(u => u.ImageId.HasValue).Select(u => u.ImageId.Value).Distinct().ToList(),
```
Comment (MariyaSin) [inline src/DepartmentService.Business/Department/GetDepartmentCommand.cs:348]: i think no need use distinkt here

**Pull request closed without merging**